tests/driver_encx24j600/Makefile

```diff
@@ -5,7 +5,7 @@ FEATURES_REQUIRED = periph_spi periph_gpio
 
 BOARD_INSUFFICIENT_MEMORY := msb-430 msb-430h nucleo32-f031 nucleo32-f042 \
                              nucleo32-l031 nucleo-l053 stm32f0discovery telosb \
-                             z1
+                             wsn430-v3_1b wsn430-v1_4 z1
```

Comment:
Should be wsn430-v**1_3**b!

Reply:
Fixed (and all the other insufficient memory stuff, Murdock reported.... Why is it getting so big :sob:?)